draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -395,8 +395,29 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
 
 ### Real-Time Protocols
 
-(TODO: Clarify that real time congestion controls are included, with
-allowances for the poor documentation / open source availability of these)
+General-purpose protocols coexist in the Internet with real-time congestion
+control protocols, which in general have finite throughput requirements (i.e.
+they do not seek to utilize all available capacity) and more strict latency
+bounds.
+
+{{?RFC8868}} provides suggestions for real-time congestion control design and
+{{?RFC8867}} suggests test cases. {{?RFC9392}} describes some considerations
+for the RTP Control Protocol (RTCP). This document does not change the
+informational status of those RFCs.
+
```

Comment:
Feedback for congestin control  for real-time flows can be diifferent and less frequent than the acknowledgements provided by reliable transports, RFC 9392 provides some guidance on how to design and evaluate appropriate feedback.